README.md

```diff
@@ -24,36 +24,29 @@ Worker also expands the set of tables that quartz uses with additional tables to
 5. Run Piezo as specified in [Running](#running).
 
 ###Building
-You must have [sbt](http://www.scala-sbt.org/) version 0.13.0 or higher to build the worker project.
+You must have [sbt](http://www.scala-sbt.org/) version 0.13.0 or higher installed.
 
-To compile the project run
+From the [worker](worker) directory,
 
-`make compile`.
```

Comment:
Why do you remove make? It is usually recommended to have in an open source project as it is much more universal than sbt.